go.mod

```diff
@@ -3,89 +3,37 @@ module github.com/earthly/earthly
-	github.com/godbus/dbus v4.1.0+incompatible // indirect
 	github.com/golang/protobuf v1.3.5
-	github.com/google/pprof v0.0.0-20191025152101-a8b9f9d2d3ce // indirect
-	github.com/gorilla/handlers v0.0.0-20150720190736-60c7bfde3e33 // indirect
-	github.com/hashicorp/errwrap v0.0.0-20141028054710-7554cd9344ce // indirect
-	github.com/hashicorp/go-multierror v0.0.0-20161216184304-ed905158d874 // indirect
-	github.com/hashicorp/yamux v0.0.0-20200609203250-aecfd211c9ce
-	github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 // indirect
-	github.com/jmespath/go-jmespath v0.0.0-20160803190731-bd40a432e4c7 // indirect
 	github.com/joho/godotenv v1.3.0
-	github.com/marstr/guid v1.1.0 // indirect
-	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
-	github.com/mitchellh/mapstructure v1.1.2 // indirect
-	github.com/mitchellh/osext v0.0.0-20151018003038-5e2d6d41470f // indirect
 	github.com/moby/buildkit v0.7.1-0.20200708233707-488130002abb
-	github.com/ncw/swift v1.0.47 // indirect
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.1
-	github.com/opencontainers/runtime-tools v0.0.0-20181011054405-1d69bd0f9c39 // indirect
 	github.com/otiai10/copy v1.1.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v0.0.0-20180209125602-c332b6f63c06 // indirect
-	github.com/prometheus/common v0.0.0-20180110214958-89604d197083
 	github.com/prometheus/procfs v0.0.5 // indirect
-	github.com/satori/go.uuid v1.2.0 // indirect
 	github.com/sirupsen/logrus v1.6.0
-	github.com/smartystreets/goconvey v0.0.0-20190330032615-68dc04aab96a // indirect
-	github.com/spf13/cobra v0.0.3 // indirect
 	github.com/stretchr/testify v1.5.1
-	github.com/urfave/cli v1.22.2
 	github.com/urfave/cli/v2 v2.1.1
-	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f // indirect
-	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
-	github.com/xeipuuv/gojsonschema v0.0.0-20180618132009-1d523034197f // indirect
-	github.com/yvasiyarov/go-metrics v0.0.0-20140926110328-57bccd1ccd43 // indirect
-	github.com/yvasiyarov/gorelic v0.0.0-20141212073537-a9bba5b9ab50 // indirect
-	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20140908184405-b21fdbd4370f // indirect
-	golang.org/x/arch v0.0.0-20190927153633-4e8777c89be4 // indirect
 	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
-	golang.org/x/lint v0.0.0-20190930215403-16217165b5de // indirect
-	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45 // indirect
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	google.golang.org/api v0.0.0-20160322025152-9bf6e6e569ff // indirect
-	google.golang.org/cloud v0.0.0-20151119220103-975617b05ea8 // indirect
-	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/kubernetes v1.13.0 // indirect
 )
 
 replace (
+	github.com/alessio/shellescape => github.com/alexcb/shellescape v0.0.0-20200921195046-bf72418e9bfb
```

Comment:
What's the reason for the fork?

Reply:
waiting for my PR to get merged: https://github.com/alessio/shellescape/pull/6